### ISSUE TITLE
No OAuth

### ISSUE THREAD

**Issue by AdamEsterle** [Dwolla/dwolla-swagger-php#6]
I could not find the code to do an OAuth initiation for Dwolla Direct.

https://docsv2.dwolla.com/#request-user-authorization
Shows to use this code

```
$OAuth = new Dwolla\OAuth();
$OAuth->settings->client_id = $apiKey;
$OAuth->settings->client_secret = $apiSecret;

$url = $OAuth->genAuthUrl("https://myredirect.com/redirect");
```

But that seems outdated and part of [dwolla-php](https://github.com/Dwolla/dwolla-php) (API v1)

dwolla-swagger-php has OAuthResponse, but that's it

Any help?

**Comment by AdamEsterle** [comment]
Yes, also the "Application" access token. So both the user OAuth and the Application access tokens

**Comment by dharabhatia** [comment]
I want to get Access token using "Application access token" how will I do in API V2 dwolla swagger. Which class I will used to get it. Any idea please help me out. 
https://docsv2.dwolla.com/#application-access-token

Thanks

**Comment by ShreyaThapa** [comment]
Closing this issue as it was opened in 2016 and is no longer relevant to the current version of the API and SDK. Please refer to the latest OAuth documentation for authentication: [Dwolla OAuth Documentation](https://developers.dwolla.com/docs/balance/api-reference/api-fundamentals/making-requests-and-authentication#authentication). If you encounter issues with the current version, feel free to open a new issue.
